### DISCUSSION TITLE
OntoNotes 5.0 dependency parsing

### DISCUSSION THREAD

**Discussion by danielvasic** [explosion/spaCy#8501] [Help: Other Questions] [answered]
Hello everyone, 

I have a question about dependency parsing dataset that is used in spaCy. Is the dataset for dependency parsing publicly available anywhere? I found that the dataset is OntoNotes 5.0 but after converting with StanfordCoreNLP parser in conllu format. If the dataset is not available would it be possible to release the dataset publically.

**Comment by polm** [comment]
OntoNotes is available for a fee from the LDC. 

https://catalog.ldc.upenn.edu/LDC2013T19

It is "public" in the sense that it is available to anyone. 

We can't release it the same way we do the spaCy source due to rights issues.

**Reply by danielvasic** [reply]
Thanks for the response :-)

Of course I understand, I was actually asking about dependency part of the corpus, as the OntoNotes in its original version contains only constituency parse tree.

**Comment by adrianeboyd** [comment, marked as answer]
And to clarify a bit, it's OntoNotes 5 but converted with the ClearNLP converter, with some additional work to align OntoNotes/PTB in order to have raw texts for as many documents as possible. Some of the source/info links were accidentally left out of the v3.0.0 model metadata, but they'll be added back in v3.1.0. In the meanwhile, you can see the v2 info here (it's all the same): https://v2.spacy.io/api/annotation#dependency-parsing

**Reply by danielvasic** [reply]
Dear @adrianeboyd ,

Thanks for the response and the clarification I will try the script for the conversion. I only need the test set for the evaluation so I suppose I don't have much work on the alignment part?

**Reply by adrianeboyd** [reply]
It still affects the evaluation because gold tokenization will inflate all the other measures very slightly, but it shouldn't make that much of a difference because the tokenizer is already at 99.9%. Also be aware that our reported evaluations are on the dev set, not the test set.

**Reply by danielvasic** [reply]
Dear @adrianeboyd,

Yes I understand, I'm more intrested in error cases of dependency parser anyway, the precision itself is not very important to me.

Beat regards :-)
